figures/scripts/fig3-chromothripsis-barplot.R

```diff
@@ -144,3 +151,9 @@ ggsave(file.path(output_dir, "chromothripsis_prop_cancer_group.pdf"),
        plot = bar_plot,
        height = 7,
        width = 9)
+
+# Write CSV file for Zenodo upload
+# Note that this plot is summarized without sample information to arrange on
+readr::write_csv(chromoth_cancer_group_df, fig3d_csv)
+
+chromoth_cancer_group_df
```

Comment:
Guessing we don't need to write out the df
```suggestion

```

Reply:
yeah, this was probably a dev thing..